Steps4Impact/Networking/AKFCausesService.swift

```diff
@@ -149,7 +149,10 @@ class AKFCausesService: Service {
     shared.request(.patch, endpoint: .participant(fbId: fbid),
                    parameters: JSON(["team_id": "null"]), completion: completion)
   }
-
+  static func editTeamName(team: Int, name: String, completion: ServiceRequestCompletion? = nil) {
```

Comment:
Should this be `editTeamName` or `setTeamName` perhaps?  This allows you to set the team name, which is the "edit"

Reply:
I think going with edit is fine - set implies it wasn't set before - which it was during team creation